Resources/Prototypes/Roles/Jobs/Engineering/technical_assistant.yml

```
@@ -1,10 +1,12 @@
-﻿- type: job
+- type: job
```

I'm just trying to see how this line was even touched...

---

```suggestion
- type: job 
```
oh there's supposed to be a space after it

---

wait shit no there isn't what the hell man

---

Fuck it we ball